mlflow/keras.py

```diff
@@ -23,16 +23,21 @@
 from mlflow.utils.model_utils import _get_flavor_configuration
 
 FLAVOR_NAME = "keras"
+_CUSTOM_OBJECTS_SAVE_PATH = "custom_objects.cloudpickle"
```

Comment:
Nit: Can we add inline comments documenting the purpose of these module-level strings variables?